test/sql/include/ddl_ops_1.sql

```diff
@@ -31,7 +31,7 @@ SELECT * FROM create_hypertable('"public"."Hypertable_1"', 'time', 'Device_id',
 SELECT * FROM create_hypertable('"customSchema"."Hypertable_1"', 'time', NULL, 1, chunk_time_interval=>_timescaledb_internal.interval_to_usec('1 month'));
 
 SELECT * FROM _timescaledb_catalog.hypertable;
-SELECT * FROM _timescaledb_catalog.hypertable_index;
+SELECT * FROM _timescaledb_catalog.hypertable_index ORDER BY format('%I.%I', main_schema_name, main_index_name)::regclass;
```

Comment:
Why is the order by added ?

Reply:
so that we have more easy-to-understand output and changes in the golden file.